README.md

```diff
@@ -14,18 +14,59 @@ This solves many use cases like:
 
 # Features
 
-  - A side view to display product hierarchy.
+  - A left pane to display project hierarchy.
   - Search functionality for faster access to VIs.
```

Comment:
"Search functionality to find things of interest quickly."?

Reply:
Fixed.